docs/docs/integrations/caches/redis_llm_caching.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Redis Cache for LangChain\n",
```

Comment:
Would you mind updating the existing Redis cache documentation in [this doc](https://github.com/langchain-ai/langchain/blob/master/docs/docs/integrations/llm_caching.ipynb) instead of creating this page?

Reply:
@ccurme can you discuss with @efriis ? Thanks

Reply:
OK to add a new page, thanks.